spec/fixtures/flight_data_return.json

```diff
@@ -0,0 +1,169 @@
+			            "trip_duration":5,
+			            "id":242,
+			            "min_f":30.400000000000034,
+			            "max_f":34.99000000000001,
+			            "min_c":-1.599999999999966,
+			            "max_c":2.990000000000009,
+			            "day_feels_like_c":-7.0499999999999545,
+			            "day_feels_like_f":24.950000000000045,
+			            "description":"clear sky",
+			            "date":"2021-01-09T02:00:00+00:00",
+			            "chance_precip":20,
+			            "sky_coverage":20,
+			            "weather_data_available":true
+			          }
+			        },
+							{
+				        "id":"242",
+				        "type":"trip",
+				        "attributes":
+				          {
+				            "origin_city":"Denver",
+				            "destination_city":"Houston",
+				            "origin_iata":"DEN",
+				            "destination_iata":"IAH",
+				            "latitude":29.9844444,
+				            "longitude":-95.341389,
+				            "price":49,
+				            "departure_datetime":"2021-01-30T10:07:00.000+00:00",
+				            "arrival_datetime":"2021-01-30T13:32:00.000+00:00",
+				            "r_departure_datetime":"2021-02-04T14:41:00.000+00:00",
+				            "r_arrival_datetime":"2021-02-04T16:19:00.000+00:00",
+				            "booking_link":
+				            "https://www.kiwi.com/deep?from=DEN&to=IAH&flightsId=0713230e48e1000055702f9c_0%7C230e071348e6000063003075_0&price=40&passengers=1&affilid=picky&lang=en&currency=USD&booking_token=BH6d3ZbUWgOg05ck0nutzMR4nWvyYNTeEM_aYAd77NEcM60o7L5uTwn1n3qwVJJSXxQHxqIz78x2u1OxJ0GUu4sfBcHo8HjNSsF9H4Vai0ikkuKhcANGpQhfH-JUuXVibvGRT3DpV1MlmLQ7KSomq_gzNK3squKcjEoMh4xoG8zcpYlPFgVXRHEpBB1U-elSTs0tq9IPMmqPLTPuYP-PZfQxUcwyO3SSWEr1P5N1P24UpGqcyTLSc_RfzwnIj0xe4JAYifMo9eqtftdHuMQleLmgL6JNAYNcRJGeEitGpj8ScrCV-dHmybWwNdJcZOL_HtS8sY8aKH9SC9Rm0spEZ63VLPX_dGZ40s6BPR-Xc294MijvJi5f9l8IspbxGQqx6hpcYXcXrJeP1Hkva3Ckj6cXdNsUE0lbG-cDFbsMT_us9GBRpeuBx3MFYmZiGaD8BDj7QvuInX9KLexxdjgul-DTbx3AruY0U1cx3_X-CiQuBDUHAJXcJyY7d9PuRnoKCH15-YuwJaMGx7ykEKHy84iieVT_S9Uv-4R8ElVEUAXfnrFch1CANNx3D02B0oQRp-N3g-09jEQGmLvKd-d1Zcia3M3mpVz1VDdyfyy7_X8AkSx5x02LK0MiESKqV2_EFWGobFuw0GrYScsvJ_XuLgrmeWVhQszHfDKuYNPXo71U=",
+				            "trip_duration":5,
+				            "id":242,
+				            "min_f":30.400000000000034,
+				            "max_f":34.99000000000001,
+				            "min_c":-1.599999999999966,
+				            "max_c":2.990000000000009,
+				            "day_feels_like_c":-7.0499999999999545,
+				            "day_feels_like_f":24.950000000000045,
+				            "description":"clear sky",
+				            "date":"2021-01-09T02:00:00+00:00",
+				            "chance_precip":20,
+				            "sky_coverage":20,
+				            "weather_data_available":true
+				          }
+				        }
+    ]
```

Comment:
Nice on this fixture file! I think that the `weather_data_available` can be removed.

Reply:
Awesome work on this.  I realized some typos on the weather api readme (i made) - we are not currently we're not passing back either 'chance_precip' or 'sky_coverage' from the weather api.  I'm not sure if those are even fields in the api.

Reply:
This has been updated. Thanks!